### PR TITLE
Update readme examples

### PR DESCRIPTION
**Pull request by DamjanOstrelic**
Hi, please keep in mind I am totally new to Go 😄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,13 +146,14 @@ func main() {
       defer wg.Done()
       // switching host. all options changes are safe to call from goroutines.
       koios.Host(host)(api)
-      res, _ := api.GET(ctx, "/tip")
+      res, _ := api.GET(ctx, "/tip", make(map[string][]string), make(map[string][]string))
```
Comment (mkungla) [inline README.md:149]: this should be `api.GET(ctx, "/tip", nil, nil)`

**Review by mkungla — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (DamjanOstrelic) [reply README.md:149]: 👍 thx

Comment (mkungla) [reply README.md:149]: so if you send another commit with that changed it can be merged

Comment (DamjanOstrelic) [reply README.md]: done

**Review by mkungla — APPROVED** [review]

**Pull request merged**